### PR TITLE
Add Notifi to Notifications Switch Modal

### PR DESCRIPTION
**Pull request by jennifer-notifi**
- Add Notifi to Notifications Switch Modal
- Add Announcements List to the Modal

## **Existing User**
https://www.dropbox.com/s/fpkaif8jrokdiba/FInalVideo.mov?dl=0

## **New User**:
https://www.dropbox.com/s/hl4a7dw0w5d5759/New%20User.mov?dl=0

### CODE REVIEW

File: src/components/NotifiNotifications/Announcements.jsx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            </TimelineContent>
+          </TimelineItem>
+        );
+      })}
+    </Timeline>
+    </>
+    )
+}
+
+const useStyles = makeStyles((theme) => ({
```
Comment (tradefortendies) [inline src/components/NotifiNotifications/Announcements.jsx:116]: Usually `styles` goes to top of the page.

**Review by 0xM4m4n — APPROVED** [review]
Looks good to me

**Comment by tradefortendies** [comment]
@jennifer-notifi would we still have the announcements page?

**Review by thedonmon — APPROVED** [review]
Looks good

**Comment by jennifer-notifi** [comment]
> 

Hello @tradefortendies, we added an announcements modal if users subscribe to Notifi and removed the announcements page. Would you like us to keep the announcements page?

File: src/components/NotifiNotifications/Announcements.jsx (hunk 1 above)
Comment (jennifer-notifi) [reply src/components/NotifiNotifications/Announcements.jsx:116]: Thank you for the note! Thought adding it at the end would help with readability, didn't know that should have it at the top! Will update on my end :)

**Comment by tradefortendies** [comment]
> > 
> 
> Hello @tradefortendies, we added an announcements modal if users subscribe to Notifi and removed the announcements page. Would you like us to keep the announcements page?

this makes sense

**Pull request merged**